code/game/objects/items/weapons/design_disks/oberth_republic.dm

```diff
@@ -21,4 +21,28 @@
 		/datum/design/autolathe/ammo/srifle,
 		/datum/design/autolathe/ammo/srifle/practice = 0,
 		/datum/design/autolathe/ammo/srifle/rubber,
+	)
+
+/obj/item/computer_hardware/hard_drive/portable/design/guns/sts
+	disk_name = "Oberth Republic - .30 STS Assault Rifle"
+	icon_state = "black"
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED
+	rarity_value = 65
+	license = 12
+	designs = list(
+		/datum/design/autolathe/gun/sts35 = 3, // "OR STS"
+		/datum/design/autolathe/ammo/lrifle,
+		/datum/design/autolathe/ammo/lrifle/practice = 0,
+		/datum/design/autolathe/ammo/lrifle/rubber,
```

Comment:
I really don't see why assault rifles should have rubbers in the design disk, especially alongside lethals.

Reply:
Cry about it.